core/domain/exp_services.py

```diff
@@ -1622,6 +1623,43 @@ def get_next_page_of_all_non_private_commits(
     ) for entry in results], new_urlsafe_start_cursor, more)
 
 
+def get_image_filenames_from_exploration(exploration):
+    """ Get the image filenames from the exploration.
+
+    Args:
+        exploration: object. The exploration itself.
+
+    Returns:
+       list. The list containing the name of the image files in the exploration
+    """
+    filenames = []
+    for state in exploration.states.itervalues():
+        if state.interaction.id == 'ImageClickInput':
+            if isinstance(state.interaction.customization_args['imageAndRegions']['value']['imagePath'], dict): # pylint: disable=line-too-long
```

Comment:
Goodness, no -- this is an abuse of `pylint: disable`. Use intermediate named variables if you need them. There's no point having lint checks if people just try to go around them.

/cc @apb7

Reply:
Done!